src/Microsoft.IdentityModel.Logging/IdentityModelTelemetryUtil.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
```

Comment:
copyright header

Reply:
Aside: header can be shortened to: 
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

see e.g.: [https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/IAccount.cs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/master/src/client/Microsoft.Identity.Client/IAccount.cs)

A change for another PR / Work Item